src/Trakx.CoinGecko.ApiClient/CoinGeckoClients/CoinGeckoClient.cs

```diff
@@ -40,4 +42,9 @@ private async Task<T> GetFromCacheOrApi<T>(string cacheKey, Func<Task<T>> getFro
 
         return value!;
     }
+
+    internal string BuildCacheKey(params object?[] keyFragments)
+    {
+        return string.Join('|', keyFragments.Prepend(_typeName));
```

Comment:
[suggestion] should we use _ instead of | ??

Reply:
https://github.com/search?q=org%3Atrakx+cacheKey+language%3AC%23&type=code&l=C%23

We use |, _ and -, so there's no standard.
I personally prefer the pipes because it's a more uncommon char.